docs/pipelines/training.md

```diff
@@ -220,7 +220,7 @@ run_cmd(
     cluster=cluster,
     expname=f"{expname}-packing",
     container="nemo", # please use "nemo container" for packed data prepration
-    partition="cpu",  # if available on your cluster
+    partition="cpu",  # if available on your cluster, but we suggest you use gpu partition="interactive" if you have large dataset, e.g., 200k
```

Comment:
```suggestion
    # this is a cpu-only operation, so if a cluster has a good cpu partition, it can be used
    # note that this is an expensive operation requiring a lot of CPUs and RAM
```